pkgs/applications/misc/gopacked/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gopacked";
+  version = "0.4.1";
+
+  subPackages = [ ];
```

Comment:
What happens if you remove this?

Reply:
Works perfectly without "SubPackages" = fixed!